OneCommunityGlobal/HighestGoodNetworkApp
# Yongjian - Fix minor alignment of assign team button on teams tab

# Description
![Screen Shot 2023-06-01 at 8 00 31 PM](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/46613683/1b236cb4-b2bb-4f13-9438-0232ff2933ce)

## Mainly changes explained:
Adjusted width of "Assign Team" button under "Teams Tab" under user profile page.

## How to test:
1. check into current branch
2. do `npm run start:local` to run this PR locally
3. login as admin role -> user profile under dashboard -> Teams Tab -> inspect the change in width of the "Assign Team" button.

## Screenshots
![Screen Shot 2023-06-01 at 8 01 42 PM](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/46613683/26bbc3d3-cb12-4fe9-96e8-9b1f202917c8)
